example/scripts/test_example.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+cleanup() {
+  killall -q test_replica || true
+  killall -q minio || true
+  rm -rf gen-sec.*
+  rm -rf exampleReplicaTests_DB_*
+}
+
+#trap 'cleanup' SIGINT
+
+cleanup
+
+scriptdir=$(cd $(dirname $0); pwd -P)
+echo $scriptdir
+
+echo "Generating new keys..."
+$scriptdir/../../tools/GenerateConcordKeys -f 1 -n 4 -o replica_keys_
```

Comment:
Are we considering number of replicas to be configurable in future?

Reply:
So as an initial commit, I have added an test_example.sh script to run client and replica but it will not be like this and will add a python script which will responsible to run this example/demo and user can easily configure the parameters using that script.